frontend/packages/react-app/src/reducers/token.ts

```diff
@@ -0,0 +1,20 @@
+export interface TokenInfo {
+  token: {
+    name: string;
+    tokenAddress: string;
+  };
+}
+
+export type ACTIONS = {
+  type: "token";
```

Comment:
Action type should contain what action do actually. 
So in this case it can be like `SET_TOKEN`. 
https://redux.js.org/style-guide/style-guide#write-action-types-as-domaineventname

Reply:
OK!

Reply:
Additionally commented here and we should decide style. 
https://github.com/TART-tokyo/iroiro/pull/63#discussion_r530753450

Reply:
I added comment.
Let's use `domain:action` style.
https://github.com/TART-tokyo/iroiro/pull/63#discussion_r531044630